### PR TITLE
[Remove Spring extender usages] LPS-159285

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to remove spring extender usage in `commerce-machine-learning-forecast-alert-service`, thanks!
https://issues.liferay.com/browse/LPS-159285

### CODE REVIEW

File: modules/dxp/apps/commerce/commerce-machine-learning-forecast-alert-service/src/main/java/com/liferay/commerce/machine/learning/forecast/alert/service/impl/CommerceMLForecastAlertEntryServiceImpl.java [hunk 1]
```diff
@@ -171,7 +182,7 @@ private long[] _getUserCommerceAccountIds(long userId)
 				"_portletResourcePermission",
```
Comment (dantewang) [inline modules/dxp/apps/commerce/commerce-machine-learning-forecast-alert-service/src/main/java/com/liferay/commerce/machine/learning/forecast/alert/service/impl/CommerceMLForecastAlertEntryServiceImpl.java:182]: @JanisZhang Please also migrate this resource permission to the OSGi reference way.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
force-pushed based on the comment.

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:acceptance

**Comment by JanisZhang** [comment]
Hi @catarineaguiar , Could you review the test results when they finish? Thank you!

**Review by dantewang — APPROVED** [review]
@zhangyan0701 Could you check the test results?

@tinatian Code looks good to me, asked QA to check the results.

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
ci:forward

**Pull request closed without merging**